app/services/userService.js

```diff
@@ -26,25 +27,31 @@ exports.comparePassword = async (password, dbPassword) => {
 
 exports.findUserByEmail = email => UserModel.findOne({ where: { email: formatEmail(email) } });
 
-exports.createUser = async ({ firstName, lastName, email, password }) => {
+exports.createUser = async ({ firstName, lastName, email, role, password }) => {
   const hash = await this.encryptPassword(password);
 
   const user = await UserModel.create({
     firstName,
     lastName,
     email: formatEmail(email),
-    password: hash
+    password: hash,
+    role
   });
 
-  const token = generateToken(user.id);
+  const token = generateToken(user);
 
   logger.info(`Usuario creado: ${user.firstName} ${user.lastName}`);
   return { user, token };
 };
 
 exports.login = user => {
-  const token = generateToken(user.id);
+  const token = generateToken(user);
```

Comment:
Is this including the password hash?

Reply:
I think so, because the user object in that moment doesn't got mapped. But in the token generation I never include the password.